docs/advanced_usage/concepts.md

```diff
@@ -34,7 +34,7 @@ hyperparameters for a Template. Hence, Pipelines:
   hyperparameters of their template.
 
 A pipeline can be fitted and evaluated directly using [MLBlocks](
-https://hdi-project.github.io/MLBlocks), or using the **GreenGuardPipeline**.
+https://MLBazaar.github.io/MLBlocks), or using the **DracoPipeline**.
```

Comment:
what is `DracoPipeline`?

Reply:
I will mark this for when we have good documentation on Draco pipelines